app/src/main/java/ca/marshallasch/veil/database/Database.java

```diff
@@ -1450,7 +1453,76 @@ public int getNumLost(@IntRange(from = 0, to = 3) int protocolVersion) {
         return getCount(SyncStatsEntry.TABLE_NAME, selection, selectionArgs);
     }
 
+    /**
+     * Updates the status for the post Where
+     *  0 = normal
+     *  1 = protected
+     *  2 = dead
+     *
+     * @param postHash the hash of the post item being marked
+     * @param status 0,1,2 as describe in the comment
+     * @return
+     */
+    @WorkerThread
+    public boolean setPostStatus(String postHash, int status) {
+        String selection = KnownPostsEntry.COLUMN_POST_HASH + " = ? AND " +
+                KnownPostsEntry.COLUMN_COMMENT_HASH + " == \"\" ";
```

Comment:
I belive that this should be a single `=` not `==`

Reply:
resovled